src/rf/apps/workers/process.py

```diff
@@ -252,16 +250,20 @@ def emr_heartbeat(self, record):
             return False
 
         try:
-            int(layer_id)
+            layer_id = int(layer_id)
         except ValueError:
             return False
 
         try:
             layer = Layer.objects.get(id=layer_id)
         except Layer.DoesNotExist:
-            return False
+            return True
```

Comment:
I had been leaving these so they would end up in the dead letter queue. Do you want to remove the message. My thought was that the dead letter queue might be helpful for debugging.